### PR TITLE
[TorchMhlo] enable binary unittests

### PR DESCRIPTION
**Pull request by fortianyou**

### CODE REVIEW

File: pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp [hunk 1]
```diff
@@ -247,6 +297,56 @@ LogicalResult torchAlphaToMhloTensor(
   return success();
 }
 
+template <typename AtenOpT, mhlo::ComparisonDirection DirectionT>
+class ConvertAtenBinaryCompareOp : public OpConversionPattern<AtenOpT> {
```
Comment (Yancey1989) [inline pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp:301]: Add a FileCheck?

File: pytorch_blade/tests/disc/ops/test_binary_ops.py [hunk 2]
```diff
@@ -84,18 +84,18 @@ def _test_binary_ops(self, binary_ops_func, test_int=True):
             out, res = self._test_cvt_to_disc(binary_ops_func, test_data)
             self._check_type(out, res)
 
-    def _test_func(self, torch_func):
+    def _test_func(self, torch_func, test_int=True):
```
Comment (Yancey1989) [inline pytorch_blade/tests/disc/ops/test_binary_ops.py:87]: #391 has merged, please add some shape annotation to test dynamic shape test case.

File: pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp (hunk 1 above)
Comment (qiuxiafei) [reply pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp:301]: +1

Comment (fortianyou) [reply pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp:301]: done

File: pytorch_blade/tests/disc/ops/test_binary_ops.py (hunk 2 above)
Comment (fortianyou) [reply pytorch_blade/tests/disc/ops/test_binary_ops.py:87]: done

**Review by Yancey1989 — APPROVED** [review]
LGTM

**Pull request merged**